integrasjon/pdl-klient/src/test/java/no/nav/vedtak/felles/integrasjon/pdl/PdlKlientTest.java

```diff
@@ -112,8 +114,26 @@ void skal_returnere_bolk_med_identer() throws IOException {
             identer.stream()
                 .flatMap(r -> r.getIdenter().stream())
                 .map(IdentInformasjon::getIdent)
-                //.collect(Collectors.toList())
+            //.collect(Collectors.toList())
         )
             .containsExactlyInAnyOrder("16047439276", "9916047439276", "25017312345", "9925017312345");
     }
+
+    @SuppressWarnings("resource")
+    @Test
+    void skal_returnere_ikke_funnet() throws IOException {
+        when(httpEntity.getContent()).thenReturn(getClass().getClassLoader().getResourceAsStream("pdl/errorResponse.json"));
+
+        var queryRequest = new HentIdenterQueryRequest();
+        queryRequest.setIdent("12345678901");
+        var projection = new IdentlisteResponseProjection()
+            .identer(
+                new IdentInformasjonResponseProjection()
+                    .ident()
+                    .gruppe()
+            );
+
+        assertThrows(FunksjonellException.class, () -> pdlKlient.hentIdenter(queryRequest, projection, Tema.OMS));
```

Comment:
Hadde kanskje vært enda bedre å assertet på at feilmeldingen inneholder "person ikke funnet"

Reply:
Jeg vil heller over fra VLException til dedikerte exceptions så man kan catche og slippe være avhengig av teksten i feilmelding høyere opp i applikasjonene.
Nå er det overskudd av TekniskException og man må sjekke på kode eller innhold for å velge håndter eller rethrow.